README.md

```diff
@@ -650,6 +693,47 @@ config.resolve
 config.resolve.plugins.delete(name)
 ```
 
+#### Config resolve plugins: ordering before
+
+Specify that the current `plugin` context should operate before another named `plugin`.
+You cannot use both `.before()` and `.after()` on the same resolve plugin.
+
+```js
+config.resolve
+  .plugin(name)
+    .before(otherName)
+
+// Example
+
+config.resolve.plugin('beta')
```

Comment:
The non-resolve examples and also the config.resolve "after" example below, put the `.plugin()` on a new line, so I think this should be too?